docs/source/poet/poet_primer.rst

```diff
@@ -0,0 +1,153 @@
+***********
+PoET Primer
+***********
+
+The Proof-of-Elapsed-Time or PoET Consensus method offers a solution to the
+Byzantine Generals Problem that utilizes a “trusted execution environment” to
+improve on the efficiency of present solutions such as Proof-of-Work. The
+initial reference implementation of PoET released to Hyperledger was written for
+an abstract TEE to keep it flexible to any TEE implementation. This
+specification defines a concrete implementation for SGX. The following
+presentation assumes the use of Intel® SGX as the trusted execution environment.
+
+At a high-level, PoET stochastically elects individual peers to execute requests
+at a given target rate. Individual peers sample an exponentially distributed
+random variable and wait for an amount of time dictated by the sample. The peer
+with the smallest sample wins the election. Cheating is prevented through the
+use of a trusted execution environment, identity verification and blacklisting
+based on asymmetric key cryptography, and an additional set of election
+policies.
+
+The following gives a rough, mid-level description of PoET 1.0 and how it
+operates. It assumes an understanding of trusted execution environments such as
+Intel® SGX as well as some background knowledge of consensus models. At a
+minimum, the reader should have a high-level understanding of the Byzantine
+Generals Problem. [1]_ Many of the specific details are omitted because they
+are documented in the comprehensive :doc:`poet_spec`.
+
+Trusted Computing Base
+======================
+
+PoET leverages a trusted execution environment (TEE) to ensure fair elections.
+PoET was first designed and implemented with Intel® SGX and makes the following
+assumptions about the trusted execution environment it depends on:
+
+1. Code and data cannot be accessed or modified once they have been loaded into
+   the TEE, except through a strict interface defined prior to loading.
+2. Attestation of the code and data loaded into the TEE is possible.
+3. Individual “platforms” have a hardware encoded identity in the form of an
+   asymmetric key pair.
+4. An external attestation service (EAS) is available which:
+
+   a. Maintains a blacklist of identities
+   b. Generates verifiable attestation reports for a given identity and code
+
+5. Random number generation is possible within the TEE.
+
+Peer Signup
+===========
+
+The following describes the PoET signup process assuming a generic TEE.
+"Application" is used to refer to untrusted code that interacts with the TEE:
+
+1. PoET is loaded into the TEE by the application.
```

Comment:
I tried not to leak concepts and language from Sawtooth into the explanation of PoET, which is why I didn't use validator. I have updated the language to use "PoET Module" to refer to the untrusted code and "PoET Enclave" to refer to trusted code.